corehq/apps/es/mappings/xform_mapping.py

```diff
@@ -140,6 +140,9 @@
                         },
                         "username": {
                             "type": "keyword"
+                        },
+                        "formLoadTime": {
+                            "type": "keyword"
```

Comment:
@AmitPhulera 
Just confirming that this is the right type for this?

This value is going to be in miliseconds.

Reply:
@AmitPhulera suggested that we think about storing this as a date field. I'm quickly finding out what the `timeStart` refers to. If it refers to the timestamp a user clicked to open a form, we could store the `formLoadTime` as a date and not a difference. The benefit of doing this is that we could later do all sorts of date queries on this field if we wanted to.

Reply:
Hey @Charl1996 
I don't believe we can do this. I believe we need the time it took to load the form. I don't think we can get that using timeStart and any other timestamp.

Reply:
Hey @Charl1996 
Whats the challenge with how it is currently?

I don't believe we can do this. I believe we need the time it took to load the form. I don't think we can get that using timeStart and any other timestamp.

Reply:
My understanding is that we are storing epoch time in `formLoadTime`, Is that correct. If so that is a essentially a date field so it should be treated as date in the mappings unless there is a strong reason to keep it as a keyword.

Reply:
@AmitPhulera 

> My understanding is that we are storing epoch time in formLoadTime

The value stored in this field is something like "230" or "190", which denotes the duration in milliseconds it took to load the form. It's not epoch time

Reply:
Ahh! I see. Feel free to discard my suggestion about storing it as date. 
Another question, why are we not storing it as `integer` then?

Reply:
Mmm, I suppose `integer` would be better, yes.